src/java.base/share/classes/java/lang/StringUTF16.java

```diff
@@ -1532,27 +1528,23 @@ public static boolean contentEquals(byte[] value, CharSequence cs, int len) {
         return true;
     }
 
-    public static int putCharsAt(byte[] value, int i, char c1, char c2, char c3, char c4) {
+    public static void putCharsAt(byte[] value, int i, char c1, char c2, char c3, char c4) {
```

Comment:
Hello Roger, can this (pre-existing) method and the other `putCharsAt` that we are changing here, be changed to package private instead of `public`? As far as I can see, these 2 are only accessed from `java.lang.AbstractStringBuilder` which resides in the same package as this class.

Reply:
Doing that in a separate issue/PR sounds good to me.